integrations/sentry/clients/sentry.py

```diff
@@ -23,6 +32,40 @@ def __init__(
         self.client = http_async_client
         self.client.headers.update(self.base_headers)
         self.selector = cast(SentryResourceConfig, event.resource_config).selector
+        self.rate_limit_remaining = MINIMUM_LIMIT_REMAINING + 1
+        self.rate_limit_concurrent_remaining = MINIMUM_CONCURRENT_REQUESTS_REMAINING + 1
+
+    async def fetch(
+        self,
+        url: str,
+        params: dict[str, Any] | None = None,
+    ) -> httpx.Response:
+        while True:
+            if (
```

Comment:
I think it would be nice to have reference and explanation of the way we handling